android/src/main/java/com/stripeterminalreactnative/ReactNativeConstants.kt

```diff
@@ -16,4 +16,7 @@ enum class ReactNativeConstants(val listenerName: String) {
     START_READER_RECONNECT("didStartReaderReconnect"),
     READER_RECONNECT_SUCCEED("didSucceedReaderReconnect"),
     READER_RECONNECT_FAIL("didFailReaderReconnect"),
+    OFFLINE_STATUS_CHANGE("didChangeOfflineStatus"),
+    PAYMENT_INTENT_FORWARDED("didForwardPaymentIntent"),
+    FORWARDING_FAILURE("didReportForwardingError"),
```

Comment:
We are a bit inconsistent here. However, I recommend that we match most of these enums and rename these three to:

```
CHANGE_OFFLINE_STATUS
FORWARD_PAYMENT_INTENT
REPORT_FORWARDING_ERROR
```

It seems like, in most cases, we just remove the "did" prefix.